### PR TITLE
[cilium-hubble] build cilium-hubble from src

### PR DESCRIPTION
**Pull request by AndreyPavlovFlant**
## Description

cilium-hubble is based on a distroless images and built from sources.

## Why do we need it, and what problem does it solve?

We'd like to base our images on a non-distro image and build everything from source.

## Checklist
- [ ] The code is covered by unit tests.
- [x] e2e tests passed.
- [ ] Documentation updated according to the changes.
- [x] Changes were tested in the Kubernetes cluster manually.

## Changelog entries

```changes
section: cilium-hubble
type: chore
summary: Distroless image for cilium-hubble
impact: cilium-hubble pods will restart
impact_level: default
```

### CODE REVIEW

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @AndreyPavlovFlant. I've started the workflow for label 'e2e/run/static'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @AndreyPavlovFlant. I've started the workflow for label 'e2e/run/openstack'...

**Comment by AndreyPavlovFlant** [comment]
/e2e/abort openstack;Standard;containerd;1.25 7686413660 7686413660-1-con-1-25 /sys/deckhouse-oss/install:pr7305 debian@5.159.101.99

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @AndreyPavlovFlant. I've recognized your '/e2e/abort' command and started the workflow...

File: modules/500-cilium-hubble/images/ui-backend/werf.inc.yaml [hunk 1]
```diff
@@ -27,10 +37,10 @@ import:
   add: /etc/ssl/certs/ca-certificates.crt
   to: /etc/ssl/certs/ca-certificates.crt
   before: install
-- artifact: {{ $.ModuleName }}/{{ $.ImageName }}-artifact
-  add: /usr/local/bin/hubble-ui-backend
+- artifact: {{ $.ModuleName }}/{{ $.ImageName }}-build-artifact
+  add: /tmp/install/backend
```
Comment (apolovov) [inline modules/500-cilium-hubble/images/ui-backend/werf.inc.yaml:41]: /go/src/github.com/cilium/hubble-ui/backend/backend pls

File: modules/500-cilium-hubble/images/ui-frontend/werf.inc.yaml [hunk 2]
```diff
@@ -1,12 +1,30 @@
+{{- $hubbleUIVersion := "0.12.1" }}
+# Source repo  settings. Delete after clone repo to fox
+{{- $_ := set . "SOURCE_REPO" "https://github.com" }}
 ---
-artifact: {{ $.ModuleName }}/{{ $.ImageName }}-base-artifact
-from: quay.io/cilium/hubble-ui:v0.12.1@sha256:e15af59a2ded739e420be82de6cbdd0ce22f8d3f00e3a10b3d2d2734e074a394
+# Based on https://github.com/cilium/hubble-ui/blob/v0.12.1/Dockerfile
+artifact: {{ $.ModuleName }}/{{ $.ImageName }}-build-artifact
+from: node:18-alpine3.18@sha256:a315556d82ef54561e54fca7d8ee333382de183d4e56841dcefcd05b55310f46
```
Comment (apolovov) [inline modules/500-cilium-hubble/images/ui-frontend/werf.inc.yaml:7]: Push the image to our registry with https://github.com/deckhouse/deckhouse/blob/main/tools/regcopy/main.go.

**Review by apolovov — CHANGES_REQUESTED** [review]

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @AndreyPavlovFlant. I've started the workflow for label 'e2e/run/openstack'...

**Review by apolovov — APPROVED** [review]

**Review by name212 — APPROVED** [review]

**Pull request merged**